Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,14 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+//console.log(tasks);
+let tasksHours = tasks.map((task)  => task.duration/60);
```

Comment:
I like that you created individual variables which hold different states of information instead of having a single variable trying to hold all values at different stages of time. However, we don't need to use `let` keyword here to create those variables. Please try to use `const` here for all variable initialisations.

Also, please remove the extra parentheses around the arguments being passed in the `map` & `filter` functions.